53/bryankimani/top_tracks.py

```diff
@@ -0,0 +1,108 @@
+import os
+import spotipy
+import spotipy.oauth2 as oauth2
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+username = input("Please enter your spotify username: ")
+artist_name = input("Please Enter Artist Name: ")
+
+
+def print_header():
```

Comment:
I see this code in various files, try to abstract it in a base class or single module

Reply:
Okay. Thanks. I will have change this